hamilton/graph.py

```diff
@@ -500,7 +500,9 @@ def _get_legend(
             node_style.update(**modifier_style)
             seen_node_types.add("materializer")
 
-        if n.tags.get("hamilton.data_loader") and "load_data." in n.name:
+        if n.tags.get("hamilton.data_loader") and (
```

Comment:
Comments + reason for this change? Confusing.

Reply:
needed because of how tags are done -- between data loaders